### PR TITLE
Move delta handlers to their own module.

### PR DESCRIPTION
**Pull request by huwshimi**

### CODE REVIEW

**Comment by huwshimi** [comment]
[Move delta handlers to their own module.](https://trello.com/c/pEkt1QOx/945-move-delta-handlers-to-their-own-module)

File: jujugui/static/gui/src/app/maraca/delta-handlers.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+const parsers = require('./parsers');
+
+/**
+  Get the consolidated updates from the deltas.
+  @param {Array} deltas - The list of deltas.
+  @returns {Object} return - The delta updates.
+  @returns {Object} return.changed - The entities to change.
+  @returns {Object} return.removed - The entities to remove.
+*/
+function getDeltaUpdates(deltas) {
```
Comment (hatched) [inline jujugui/static/gui/src/app/maraca/delta-handlers.js]: `parseDelta`  or `processDelta`

File: jujugui/static/gui/src/app/maraca/delta-handlers.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+    case 'annotation':
+      return parsers.parseAnnotation(entity);
+    default:
+      // This is an unknown entity type so ignore it as we don't know how to
+      // handle it.
+      return null;
+  }
+}
+
+module.exports = {
```
Comment (hatched) [inline jujugui/static/gui/src/app/maraca/delta-handlers.js]: Maybe you could just export the single function? Unless you forsee that you may want to export more in the future.

File: jujugui/static/gui/src/app/maraca/test-delta-handlers.js [hunk 3]
```diff
@@ -0,0 +1,120 @@
+/* Copyright (C) 2018 Canonical Ltd. */
+'use strict';
+
+const handlers = require('./delta-handlers');
+const parsers = require('./parsers');
+
+describe('Maraca', () => {
```
Comment (hatched) [inline jujugui/static/gui/src/app/maraca/test-delta-handlers.js]: this is actually a delta handlers test

**Pull request merged**